client/src/js/components/bhChangeDepot/bhChangeDepot.html

```diff
@@ -0,0 +1,12 @@
+<div uib-dropdown dropdown-append-to-body data-action="open-tools">
```

Comment:
Would it be possible to make the component just the `<a href>` part?  That way, we can add more items in the future (like requisitions) to the dropdown menu.

Alternatively, you could include a `transclude` slot so that we can add items to the menu by putting them as sub-elements of the component.
```html
<bh-change-depot>
  <a href="" new-component></a>
</bh-change-depot>
```

Personally, I prefer the first one, but it is up to you.